src/admin/models/activities.php

```diff
@@ -71,16 +119,30 @@ protected function getListQuery()
 		}
 
 		// Filter by published state
-		$published = $this->getState('filter.state');
+		$published = $this->getState('filter.published');
 
 		if (is_numeric($published))
 		{
 			$query->where('state = ' . (int) $published);
 		}
+		elseif ($published === '')
```

Comment:
@Mayaank12 
state can only be 0 OR 1

Do we need this elseif condition?

Reply:
No